content/docs/index.md

```diff
@@ -4,10 +4,10 @@
 [experiment management] tool that takes advantage of the existing software
 engineering toolset you're already familiar with (Git, your IDE, CI/CD, etc.).
 **DVC** helps data science and machine learning teams manage large datasets,
-make projects reproducible, and better collaborate.
+make projects reproducible, and collaborate better.
 
-**DVC** is available for [Visual Studio Code], any [system terminal], and as a
-[Python library].
+**DVC** is available for [Visual Studio Code], and can be installed in any
+[system terminal], and used as a [Python library].
```

Comment:
Just some copy edits here... But maybe since this PR started to adopt some ideas from [TF docs](https://developer.hashicorp.com/terraform/docs), and since it overlaps a lot with the [proposed UG index](https://dvc-org-guide-index-eegasq3qz4.herokuapp.com/doc/user-guide) (currently [Overview](https://dvc.org/doc/user-guide/overview)), we could turn this into a full-fledged "About the docs" page.